dependencymods.md

```diff
@@ -46,14 +50,15 @@ A core mod made by Homothety
 
 ## [ItemChanger](https://github.com/homothetyhk/HollowKnight.ItemChanger)
 
-A core mod made by Homothety. 
+A core mod made by Homothety.
+
 Relevant Links:
 - [Documentation](https://homothetyhk.github.io/HollowKnight.ItemChanger/)
 
 ## [MagicUI](https://github.com/BadMagic100/HollowKnight.MagicUI)
 
-A core mod made by BadMagic that simplifies creating non-menu, in-game UI elements. Unity's UI elements are low-level components, meaning you need to be
-very familiar with their implementation details to piece them together into something useful and usually requiring a significant amount of boilerplate code.
+A core mod made by BadMagic that simplifies creating non-menu, in-game UI elements. Unity's UI elements are low-level components, meaning you need to be very familiar with their implementation details to piece them together into something useful and usually requiring a significant amount of boilerplate code.
```

Comment:
I'm curious about the removal of this line break, it was intentional to keep the lines short in md, it should not be present on the published site.

Reply:
fair, though in editor you can just have wordwrap active for that

Reply:
I expect/hope most people are not wordwrapping in their IDE

Reply:
for plaintext files? there i expect the default to be word wrapping tbh